readme.md

```diff
@@ -92,4 +92,14 @@ go generate
 ## Terraform Manifest
 
 - **version** is the numeric version of the manifest format, not the version of our provider.
-- **protocol_versions** is the Terraform protocol version. This is set to 6.0 because we are using Terraform Plugin Framework to develop our provider.
\ No newline at end of file
+- **protocol_versions** is the Terraform protocol version. This is set to 6.0 because we are using Terraform Plugin Framework to develop our provider.
+
+## Debug
+- There are 4 IntelliJ IDEA run configurations in .run
+- Run "Axual build to _bin" to generate executable in /go/bin and name it terraform-provider-axual
+- Run "Axual Delve binary"(Run icon) to start Delve's debugging session on this binary. It looks for terraform-provider-axual executable in /go/bin.
```

Comment:
We need a step to install Delve.
`brew install delve`

Reply:
Done